apisix_versioned_sidebars/version-3.1.0-sidebars.json

```
@@ -0,0 +1,18 @@
+{
+  "docs": [
+    {
+      "type": "category",
+      "label": "Getting Started",
+      "items": [
+        "getting-started/get-apisix",
```

Kong 的 Path 是 `/gateway/latest/get-started/`，即 `{Product}/{Version}/{Slug}`，可以将 Get Started 作为这一章作为 `/getting-started` 的根路径下的内容，即：

```suggestion
        "getting-started",
```

但我不清楚 Docusaurus 是否支持这种写法，需要你试一试 🤔 

----

此外，Kong 用的是 `/get-started`，我们用的是 `getting-started`，需要看看其它站点是怎么用的 :)